### PR TITLE
Table selector

### PR DESCRIPTION
**Pull request by jkusa**
## Description

Refactor table selector

## Proposed Changes

- Add babel-eslint to navi-reports
- Style table selector similar to other power selects
- Use info icon for tooltips as done with metrics and dimensions

## Screenshots

Before:

Screen Shot 2019-10-29 at 8 19 40 AM

After:

table

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/addon/templates/components/navi-table-select-item.hbs [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
-  {{ember-tooltip
-    text=option.description
-    side="right"
-    popperContainer="body"
-    effect="none"
-    tooltipClass="navi-table-select-tooltip"
-  }}
-{{/if}}
\ No newline at end of file
+<div
```
Comment (balajimanoharan) [inline packages/reports/addon/templates/components/navi-table-select-item.hbs]: Do we need to make this a tagless component and define a div?

File: packages/reports/app/styles/navi-reports/components/navi-table-select.less [hunk 2]
```diff
@@ -3,40 +3,35 @@
...
+      display: flex;
       font-size: 14px;
-      height: 24px;
+      flex: 1;
+      height: 100%;
 
-      .ember-tooltip-target {
-        margin-left: 18px;
+      &[aria-expanded='true'] {
+        border: 1px solid #aaaaaa;
```
Comment (balajimanoharan) [inline packages/reports/app/styles/navi-reports/components/navi-table-select.less]: Can we switch this to one of our existing grays?

Comment (jkusa) [reply packages/reports/app/styles/navi-reports/components/navi-table-select.less]: This is the same style as power select, not our custom gray

File: packages/reports/addon/templates/components/navi-table-select-item.hbs (hunk 1 above)
Comment (jkusa) [reply packages/reports/addon/templates/components/navi-table-select-item.hbs]: This is technique going forward. Glimmer components do not have a `tagName`.

> Glimmer components shift templates to be treated as "outerHTML". There is no implicit div. All the DOM elements created by the renderer are in a template.

File: packages/reports/addon/templates/components/navi-table-select-trigger.hbs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="navi-table-select-trigger">
```
Comment (balajimanoharan) [inline packages/reports/addon/templates/components/navi-table-select-trigger.hbs]: ©️ missing

Comment (balajimanoharan) [inline packages/reports/addon/templates/components/navi-table-select-trigger.hbs]:
```suggestion
{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
<div class="navi-table-select-trigger">
```

**Pull request merged**